test/second_q/hamiltonians/test_electronic_energy.py

```diff
@@ -40,11 +35,10 @@ def setUp(self):
         super().setUp()
         driver = PySCFDriver()
         self.prop = cast(ElectronicEnergy, driver.run().hamiltonian)
-        self.prop.get_electronic_integral(ElectronicBasis.MO, 1).set_truncation(2)
 
     def test_second_q_op(self):
         """Test second_q_op."""
-        op = self.prop.second_q_op()
+        op = self.prop.second_q_op().chop()
```

Comment:
Note to self: this may no longer be needed as the `.chop()` should be part of `Fermionic.from_polynomial_tensor()`